packages/playground/src/weblets/jenkins.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+  <weblet-layout
+    ref="layout"
+    :cpu="solution?.cpu"
+    :memory="solution?.memory"
+    :disk="solution?.disk"
+    :ipv4="ipv4"
+    :dedicated="dedicated"
+    :SelectedNode="selectionDetails?.node"
+    :valid-filters="selectionDetails?.validFilters"
+    title-image="images/icons/jenkins.png"
+  >
+    <template #title>Deploy a Jenkins Instance </template>
+    <d-tabs :tabs="[{ title: 'Config', value: 'config' }]">
+      <input-validator
+        :value="name"
+        :rules="[
+          validators.required('Name is required.'),
+          validators.IsAlphanumericExpectUnderscore('Name should consist of letters ,numbers and underscores only.'),
+          name => validators.isAlpha('Name must start with alphabet char.')(name[0]),
```

Comment:
```suggestion
         (name: string) => validators.isAlpha('Name must start with an alphabetical character.')(name[0]),
```


as we agreed on [#3287 ](https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3287#discussion_r1730291955)

Reply:
please do it also on https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3338/files#diff-fc3e26378b57a55a0c7e60ae1fed09033c71b6acc8f96d0a85a85e5e00b10d4cR36 as well

Reply:
> please do it also on https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3338/files#diff-fc3e26378b57a55a0c7e60ae1fed09033c71b6acc8f96d0a85a85e5e00b10d4cR36 as well

@amr-yassir